src/big/big_rational.cr

```diff
@@ -170,6 +170,23 @@ struct BigRational < Number
     BigRational.new { |mpq| LibGMP.mpq_neg(mpq, self) }
   end
 
+  # Raises the rational to the *other*th power
+  #
+  # This will raise an exception if rational is 0 and other is negative.
```

Comment:
```suggestion
  # This will raise `ArgumentError` if rational is 0 and *other* is negative.
```

Reply:
Actually, it raises a `DivisionByZeroError`

Reply:
edited